test/unit/TicketBroker.js

```diff
@@ -1735,4 +1735,93 @@ contract("TicketBroker", accounts => {
             assert.equal(event.returnValues.reserve.toString(), reserve.toString())
         })
     })
+
+    describe("claimableReserve", () => {
+        it("returns 0 when the reserveHolder does not have a reserve", async () => {
+            const numRecipients = 10
+            await fixture.bondingManager.setMockUint256(functionSig("getTranscoderPoolSize()"), numRecipients)
+            assert.equal((await broker.claimableReserve(constants.NULL_ADDRESS, constants.NULL_ADDRESS)).toString(), "0")
+        })
```

Comment:
Style nit: Let's add a space between the `it` blocks to remain consistent with the style in the rest of the file (we should add this type of check to the JS linter at some point). Comment holds for the rest of the `it` blocks for the changes in this file as well.